src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+/**
+ * Represents a Person's contact in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidContact(String)}
+ */
+public class Contact {
+
+    public final String value;
```

Comment:
Is `public` the best access modifier for `value` ? As a rule of thumb start all data members as private, and depending on their usage promote them to protected, and if absolutely necessary, to public.